src/sinks/mod.rs

```diff
@@ -116,7 +117,6 @@ pub enum HealthcheckError {
     UnexpectedStatus { status: ::http::StatusCode },
 }
 
-/*
```

Comment:
Number of stray block comment pieces floating around in this file

Reply:
Yeah, I specifically commented out sinks that I hadn't yet converted, but didn't want to group them all together and lose my alphabetical ordering. :P